src/RigidBody/Joints.cpp

```diff
@@ -33,7 +34,7 @@ biorbd::rigidbody::Joints::Joints() :
     m_nbQddot(std::make_shared<unsigned int>(0)),
     m_nRotAQuat(std::make_shared<unsigned int>(0)),
     m_isKinematicsComputed(std::make_shared<bool>(false)),
-    m_totalMass(std::make_shared<double>(0))
+    m_totalMass(std::make_shared<biorbd::utils::Scalar>())
```

Comment:
The `0`in the parentheses is missing (`m_totalMass(std::make_shared<biorbd::utils::Scalar>(0))`)

Reply:
Done